src/Components/PresentationalComponents/Filters/PrimaryToolbarFilters/CvssBaseScoreFilter.js

```diff
@@ -1,7 +1,7 @@
 import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
 import { intl } from '../../../../Utilities/IntlProvider';
 import messages from '../../../../Messages';
-import CheckboxCustomFilter from '../CustomFilters/CvssCustomFilter';
```

Comment:
This doesn't change anything, it's just a default import rename to fix copy-paste mistake.